build/ci/job-template.yml

```diff
@@ -10,7 +10,7 @@ parameters:
 jobs:
   - job: ${{ parameters.name }}
     ${{ if eq(parameters.codeCoverage, 'false') }}:
-      timeoutInMinutes: 40
+      timeoutInMinutes: 60
```

Comment:
We should really be careful about letting our builds take so long. This isn't a huge project, and having an hour CI time doesn't seem great. Maybe someone should investigate what is taking so long in our builds.

Reply:
Eric, I will revert this back after the release. We have a huge backlog of PRs that is threatening the release.

---
In reply to: [339138790](https://github.com/dotnet/machinelearning/pull/4377#discussion_r339138790) [](ancestors = 339138790)

Reply:
Agreed - we need to get unblocked in the short term. I meant to say that someone should investigate it after this gets us unblocked.

Reply:
I suspect the reason our build times went up is because of newly added TensorFlow tests that each take easily a minute to run. We can try to reduce the dataset size but obviously after this release.

CC: @ashbhandare @harshithapv @bpstark